daemon/create.go

```diff
@@ -46,7 +54,7 @@ func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hos
 }
 
 // Create creates a new container from the given configuration with a given name.
-func (daemon *Daemon) Create(config *runconfig.Config, hostConfig *runconfig.HostConfig, name string) (retC *Container, retErr error) {
+func create(daemon *Daemon, params *ContainerCreateConfig) (retC *Container, retErr error) {
```

Comment:
The only place it's called from is within the same file. Doesn't need to be an externally visible function.